backend/services/deployments/tests/tests/conftest.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import threading
+
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from typing import Any
+
+
+class OptimistHandler(BaseHTTPRequestHandler):
+    """
+    OptimistHandler answers 200 OK to all requests.
+    """
+
+    def _handle_request(self):
+        self.send_response(200)
+        self.end_headers()
+
+    def __getattribute__(self, name: str, /) -> Any:
+        if name.startswith("do_"):
+            return self._handle_request
+        return super().__getattribute__(name)
```

Comment:
could you leave some comment about this method, I have no idea why it is needed here.

Reply:
I added a docstring for the method.